src/adapter/src/webhook.rs

```diff
@@ -276,6 +276,7 @@ mod test {
     use super::WebhookConcurrencyLimiter;
 
     #[mz_ore::test(tokio::test)]
+    #[cfg_attr(miri, ignore)] // error: returning ready events from epoll_wait is not yet implemented
```

Comment:
Can we reference a GitHub issue in the comment?

Reply:
This is just a missing feature in miri, I'm not sure if and when they are planning to add it, they don't have an issue for that afaik and I'm not sure if they'd appreciate having one for each missing feature (of which there are many): https://github.com/search?q=repo%3Arust-lang%2Fmiri+%22returning+ready+events+from+epoll_wait+is+not+yet+implemented%22&type=code

Reply:
Ok!